2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
-
+for (let friend of friends){
+  if (friend.age>35){
```

Comment:
by the name of the variable should this be `>=` ?

Reply:
yes, fixed it to >=